tsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+    "compilerOptions": {
+        "outDir": "dist"
```

Comment:
[fix] `target: 2018` or something that does not generate crazy state machines for async functions. Also, this explains the failure to do `instanceof`. The default value is `es3` which is ridiculous!